File: testresources/csharp/benchmark/Domain/Indirect/ViolatingFrom/CallInstanceMethodIndirect_MethodMethodToString.cs
namespace Domain.Indirect.ViolatingFrom
{
	public class CallInstanceMethodIndirect_MethodMethodToString
	{
        private Domain.Indirect.Intermediate.BackgroundService bckg;

		private string test;

		public void Case4()
        {
            bckg = new Domain.Indirect.Intermediate.BackgroundService();
			test = bckg.GetServiceOne().GetDay().ToString();
		}
	}
}

File: testresources/java/accuracy/src/domain/indirect/violatingfrom/CallInstanceMethodIndirect_MethodMethodToString.java
package domain.indirect.violatingfrom;

import domain.indirect.intermediate.BackgroundService;

public class CallInstanceMethodIndirect_MethodMethodToString {
	private BackgroundService bckg;
	private String test;
	
	public CallInstanceMethodIndirect_MethodMethodToString() {
		bckg = new BackgroundService();
	}

	public String Case4(){
		// Indirect invocation via normal method and with toString()
		test = bckg.getServiceOne().getDay().toString();
		return test;
	}
}
